minetest-gymnasium/minetest/minetest_env.py

```diff
@@ -108,32 +110,97 @@ def _free_port():
         return s.getsockname()[1]
 
 
+class Observation(TypedDict):
```

Comment:
Why TypedDict instead of dataclass?

Reply:
I sort of assumed that the observation has to be a dict with np arrays and match the observation space definition. Can it be a dataclass?